src/routes/safe/components/Apps/utils.ts

```diff
@@ -177,15 +177,8 @@ export const staticAppsList: Array<StaticAppInfo> = [
   },
 ]
 
-export const getAppsList = async (): Promise<AppData[]> => {
-  let result
-  try {
-    result = await fetchSafeAppsList()
-  } catch (error) {
-    console.error('Could not fetch remote apps list', error)
-  }
-
-  return result?.apps && result?.apps.length ? result.apps : staticAppsList
+export const getAppsList = async (): Promise<TokenListResult> => {
```

Comment:
We now have a custom exception class https://github.com/gnosis/safe-react/pull/2283. Example usage can be found here: https://github.com/gnosis/safe-react/pull/2340

Let's use it there

Reply:
Not in this function though. The UI code should catch errors. Services shouldn't silence them.

Reply:
Then I'd say we don't need this function at all, it's just a wrapper around API function that doesn't enhance anything